src/assets/scss/single-art.scss

```diff
@@ -0,0 +1,243 @@
+$primary-color: #06a5ff;
+$regular-font: 'Poppins';
+
+.single-art-banner {
+  height: 400px;
+  background-image: url(../images/single-art.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-blend-mode: multiply;
+  background-color: rgba(0, 0, 0, 0.5);
+  background-attachment: fixed;
+  padding: 50px;
+  padding-top: 150px;
+}
+
+.label {
+  position: absolute;
+  background-color: $primary-color;
+  color: #ffffff;
+  padding: 5px;
+  font-size: 13px;
+  box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.25);
+}
+
+.art-details .title {
+  padding-top: 40px;
+  color: #ffffff;
+}
+
+.clear {
+  clear: both;
+}
+
+.thumbnails {
+  margin-top: 20px;
+}
+
+.thumbnail {
+  float: left;
+  margin-right: 15px;
+  box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.25);
+}
+
+.thumbnail img {
+  height: 100px;
+}
+
+#single-art-content {
+  padding: 50px;
+}
+
+.art-left {
+  width: 65%;
+  float: left;
+}
+
+.art-right {
+  width: 30%;
+  float: right;
+}
+
+.art-reactions ul li {
+  display: inline;
+  margin-right: 40px;
+}
+
+.art-reactions ul li span {
```

Comment:
Kindly take advantage of SASS style rules nesting.
This block can go on line 65 above, without `.art-reactions ul li`